test/integration/cookbook-test/inspec/cookbook_test_spec.rb

```diff
@@ -0,0 +1,107 @@
+# Tests nginx _test_vhost without certificate chain recipe
+
+# Dependencies installed
+describe package('nginx') do
+  it { should be_installed }
+end
+
+describe service('nginx') do
+  it { should be_enabled }
+  it { should be_running }
+end
+
+describe port(80) do
+  it { should be_listening }
+end
+
+describe port(443) do
+  it { should_not be_listening }
+end
+
+conf_dir = '/etc/nginx/sites-available'
+
+describe file(::File.join(conf_dir, 'test-cookbook.osuosl.org.conf')) do
```

Comment:
Please change all of these to using the [nginx_conf resource](https://www.inspec.io/docs/reference/resources/nginx_conf/).

Reply:
I'm not sure if I'll be able to use that resource. The params come up empty any way I try to use it.

Reply:
Seems this is due to a bug in the current version of InSpec we're using. I'll work on backporting a fix and get back to you on this.

Reply:
@ramereth any luck so far?

Reply:
Unfortunately not. Let's skip getting this converted to that resource and add a TODO comment about converting to that once we upgrade.